RNDeviceInfo.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['repository']['url']
-  s.platform     = :ios, "9.0"
+  s.platforms     = { :ios => "9.0", :visionos => "1.0" }
   s.ios.deployment_target = '9.0'
   s.tvos.deployment_target = '10.0'
```

Comment:
You can remove this, specifying `platforms` is sufficient

Reply:
removed this & add tvos in platforms 👍🏻

Reply:
@gupta-ji6 did you push that commit? I still see the deployment_targets here and no tvOS in platforms ?

Reply:
@mikehardy - it was missed, pushed now. some new issue with yarn version in my system, it produced so many files .cache folder. hence, I missed that file.

<img width="498" alt="image" src="https://github.com/react-native-device-info/react-native-device-info/assets/21218732/b68add71-c9c3-4f80-be7f-defd95da6968">